network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/about_block.html

```diff
@@ -0,0 +1,20 @@
+{% extends "./base_streamfield_block.html" %}
+{% load wagtailimages_tags wagtailcore_tags %}
+
+{% block main_block_class %} tw-w-full {% endblock main_block_class %}
+
+{% block block_content %}
+    <div class="tw-flex tw-flex-col large:tw-flex-row medium:tw-space-x-6 tw-gap-10 medium:tw-gap-20 tw-py-32">
+        <div class="tw-w-full large:tw-w-1/2">
+            <picture>
+                <source media="(min-width: 992px)" srcset="{% image_url self.image "width-540" %}, {% image_url self.image "width-1080" %} 2x"  alt="{{ self.image_alt_text }}">
+                <source srcset="{% image_url self.image "width-510" %}, {% image_url self.image "width-1020" %} 2x" alt="{{ self.image_alt_text }}">
+                <img src="{% image_url self.image "width-540" %}" class="wagtail-image w-100" srcset="{% image_url self.image "width-510" %} 510w, {% image_url self.image "width-210" %} 210w, {% image_url self.image "width-540" %} 540w" alt="{{ self.image_alt_text }}"/>
+            </picture>
```

Comment:
I don't know if this is the best html syntax, but on `libraries/profile_page.html` we use:

```html
<div id="image-container" class="tw-place-self-center medium:tw-place-self-start">
            {% with profile_image_classes="tw-rounded-full tw-w-104 large:tw-w-128" %}
                {% if author_profile.image %}
                    {% image author_profile.image fill-256x256 as img_1x %}
                    {% image author_profile.image fill-512x512 as img_2x %}
                    <img srcset="{{ img_1x.url }}, {{ img_2x.url }} 2x" src="{{ img_2x.url }}" width="{{ img_2x.width|unlocalize }}" height="{{ img_2x.height|unlocalize }}"  alt="{{ img_2x.alt }}" class="{{ profile_image_classes }}" />
                {% else %}
                    <img src="{% static '_images/author-placeholder.svg' %}" width="100" height="100" alt="{{ author_profile.name }}" class="{{ profile_image_classes }}" />
                {% endif %}
            {% endwith %}
        </div>
```
to deal with multiple sized images.

Reply:
Thanks @jhonatan-lopes! I have updated my template using the provided code for reference. I was wondering if you could please give it a second look and let me know if it looks alright?

Thank you, I appreciate it 🙌